docs/en/api/FS-API.md

```diff
@@ -187,23 +212,40 @@ Users can override the default policy class in the
 [configuration file]({{ '/en/operation/Configuration.html' | relativize_url }}) at property
 `alluxio.user.block.write.location.policy.class`. The built-in policies include:
 
-* **LocalFirstPolicy (alluxio.client.block.policy.LocalFirstPolicy)**
+* [LocalFirstPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstPolicy.html)
+
+  **This is the default policy.**
+
+  > A policy that returns local host first, and if the local worker doesn't have enough availability,
```

Comment:
`A policy that returns the worker local to the client first`

i noticed we don't use `local host` anywhere, usually referred to as `localhost`.  but localhost refers to the machine rather than the worker, which i think is what this is referring to

Reply:
I copied the descriptions verbatim from the Javadoc descriptions as I felt it'd be best to have the doc & Javadoc align; but I can fix that here for now.